MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -1,50 +1,59 @@
 PODS:
-  - AWSCore (2.5.3)
```

Comment:
Would it make sense to gitignore this file since it is actually never used?

Reply:
Bitrise is actually using this to cache some of the dependencies in order to speed up this test but I just noticed it's pointing to a non-existing path. (`./MapboxNavigationTests/CocoaPodsTest/PodInstall/Pods -> ./MapboxNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock`)